pages/solutions/segment-alternative.mdx

```diff
@@ -78,7 +78,7 @@ Grouparoo’s fundamental model is built upon querying source-of-truth data that
   heading="Easily define records, no matter your data."
   width="713"
   height="387"
-  content="Grouparoo's queryable data structure makes it easy to create records -- even from historical data.  You can configure customer records from the command line on our community edition, or straight from the UI with our enterprise product."
+  content="Grouparoo's queryable data structure makes it easy to create records -- even from historical data.  You can configure customer records from the developer UI on our community edition, or straight from the app UI with our enterprise product."
```

Comment:
I think it was right before - the developer UI doesn't let you do things in the community edition... or perhaps you are including config in the "edition"?

Reply:
Yeah - I think that's what you are saying thoughout.

Reply:
Yeps!